packages/wallets/src/features/cashier/modules/DepositCrypto/components/DepositCryptoDisclaimers/DepositCryptoDisclaimers.tsx

```diff
@@ -22,7 +22,9 @@ const DepositCryptoDisclaimers = () => {
         <div className='wallets-deposit-crypto-disclaimers'>
             <InlineMessage>
                 <div className='wallets-deposit-crypto-disclaimers__content'>
-                    <p>To avoid loss of funds:</p>
+                    <strong>
```

Comment:
@aum-deriv Why didn'y you use walletText component?

Reply:
Hi @mahdiyeh-deriv,
This code is written by someone else, I had just relocated the file to a different folder. Will change it.

Reply:
@mahdiyeh-deriv was not aware about it. Will make the change 👀

Reply:
Hi @mahdiyeh-deriv,
I checked the code for the `WalletText` component and it seems we are excluding the size of `3xs` ( size: 0.8rem, line_height: 1.2rem) in it 😕. Hence, I am unable to use it.
Maybe @aizad-deriv can help me with this.
![image](https://github.com/binary-com/deriv-app/assets/125039206/2374f548-4c11-41af-9a57-300c0341dcb5)